### DISCUSSION TITLE
Manjaro and SDR++ error

### DISCUSSION THREAD

**Discussion by AlexeyR4N** [AlexandreRouma/SDRPlusPlus#571] [testing]
Hello ! I`m using manjaro linux and I can`t to run the program sdr++
If i try to run sdr++ via console , it shows one error:

[2022-01-01 12:22:10.762] [info] SDR++ v1.0.4
[2022-01-01 12:22:10.762] [info] Loading config
error: XDG_RUNTIME_DIR not set in the environment.
[2022-01-01 12:22:10.768] [error] Glfw Error 65544: Wayland: Failed to connect to display

Before,  i used Linux Mint 20.2  and with  this OS soft everything worked well . 
SDR++ is best sdr linux soft for me, I liked.

I very want to run sdr++ on Manjaro.   
Please help me, i am newbie.

**Comment by starvald** [comment]
try installing glfw-x11 instead of glfw-wayland.

**Reply by AlexeyR4N** [reply]
Thank you very much! Now sdr++ running very good.

**Comment by AlexeyR4N** [comment]
But i have one little problem else.
In list of "Source"  not have selection  "SDRplay" 

version sdr pp is 1.05 .and  packet - "libsdrplay 3.07.1-1" was installed
I can send screenshot if it`s needed
